src/helpers/normalizeColor.ts

```diff
@@ -163,6 +163,95 @@ const colorNames = {
 
 type ColorName = keyof typeof colorNames;
 
+const NUMBER = '[-+]?\\d*\\.?\\d+';
```

Comment:
Just reordering things in this file such that they're defined before being used, no need to review.